slackbot/dispatcher.py

```diff
@@ -69,24 +74,37 @@ def _on_new_message(self, msg):
         else:
             self._pool.add_task(('listen_to', msg))
 
+    def _get_bot_id(self):
+        return self._client.login_data['self']['id']
+
```

Comment:
needed to make this a separate method in order to allow monkeypatch to fake the bot ID in unit tests